lib/ferver/file_list.rb

```diff
@@ -1,4 +1,5 @@
 require 'forwardable'
+require 'byebug'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.